### PR TITLE
To Do List

### PR DESCRIPTION
**Pull request by username-forGithub**
In this part of the project adding and removing functionalities have been added.

### CODE REVIEW

File: modules/create.js [hunk 1]
```diff
@@ -80,6 +80,3 @@ export function add() {
   updateStorege(arr);
   show();
 }
```
Comment (1GORDON) [inline modules/create.js:82]: Great design @username-forGithub 🤝

- [ ] As specified in project requirements, you are supposed to use ES6 syntax in your Javascript files. Kindly format your module functions and methods in the `create`, `delete`, and `update` files to ES6 syntax.

File: modules/create.js [hunk 2]
```diff
@@ -80,6 +80,3 @@ export function add() {
...
-
-// localStorage.removeItem('flag')
-// localStorage.removeItem('arrStored')
```
Comment (1GORDON) [inline modules/create.js:85]: Great job @username-forGithub 🤝

- [ ] As specified in the project requirements, kindly make sure your data is saved in the local storage.

File: modules/create.js (hunk 1 above)
Comment (1GORDON) [inline modules/create.js:82]: Great job @username-forGithub 🤝

- [ ] As specified in the project requirements, kindly Implement a function for editing task descriptions correctly. I can't seem to edit the tasks on your design

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @username-forGithub ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly format your readme file correctly. 
readme


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by username-forGithub** [comment]
Hi

**Review by Kingstalux — APPROVED** [review]
Hi @username-forGithub,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

## Highlights :clap: 
:ballot_box_with_check: Books can be added, edited and deleted :fire: 
:ballot_box_with_check: Good job using ES6 syntax :+1: 
:ballot_box_with_check: Professional README file :100: 

Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL) Kindly remove all hardcoded array elements so that new tasks will only be added from your site :+1: 
- [ ] (OPTIONAL) Also the value of your index should start from 1 and not 0 as stated in the requirements. Kindly rectify this issue :smile: 
- [ ] (OPTIONAL) Make sure the value of the indexes of the remaining elements are updated after one element is deleted. For example, if you have 2 tasks on your list, if the first task is deleted. then the index of the second task should be updated to 1 :+1: 
- [ ] (OPTIONAL) Finally , try to implement your functionality according to the given [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Notice that to delete an element, the user clicks on the 3 dots and the delete icon appears. It would be great if your site functions exactly like that :100: 

_These are all mandatory requirements but as they were not pointed out in your first review, I'll leave them as optional but I strongly suggest you implement the various functionalities_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**